test/unit/utils/fs.spec.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { extractTarBall, exists, rm } from '../../../src/utils/fs';
+
+describe('Tests filesystem utilities', () => {
```

Comment:
usually we only add describe blocks for each functions now

Reply:
I think, we add one upper level describe for the file for which the tests are written including describe for each functions. Ref: https://github.com/LiskHQ/lisk-service/blob/feature/1157-prepare-lisk-service-v070-for-beta/services/blockchain-connector/tests/functional/fs.test.js

Reply:
we used to that before and is done this way for integration tests as well. Actually, this adds unnecessary padding as the file name is still visible output without the describe block.